src/components/editProfile/EditForm.tsx

```diff
@@ -0,0 +1,54 @@
+import CbrumIcon from 'components/common/inside/cbrum/cbrumIcon'
+import TextInput from 'components/common/inside/forms/textInput'
+import editProfileService from 'services/users/editProfileService';
+
+import 'components/editProfile/EditForm.css'
+import rightArrow from 'assets/images/arrow_left.png'
+import { useState } from 'react';
+
+const editProfileFunction = (email: string, password: string, newPassword: string, confirmNewPassword: string) => {
+  const res = editProfileService(email, password, newPassword, confirmNewPassword);
+
+  res instanceof Error? new Notification("Profile NOT updated!") : new Notification("Profile updated!");
+}
+
+const EditForm = () => {
+  const [email, setEmail] = useState<string>("");
```

Comment:
When the type of either `string`, `boolean` or `number` you don't need to add it when using `useState`. Typescript will infer it easily
```suggestion
  const [email, setEmail] = useState("");
```